src/components/main-page/cardCVCNumberInputs/CardCVCNumberInputs.tsx

```diff
@@ -0,0 +1,43 @@
+import Input from '../../common/input/Input';
+import { CardCVCNumberSectionProps } from '../../../types/index.types';
+import { StyledContainer, StyledInputWrap, StyledErrorMessage } from '../../../styled-component/inputs';
+import { isValidNumber } from '../../../util/validation';
+import { ValidationProps } from '../../../hooks/useValidation';
+
+const CVC_NUMBER_LENGTH = 3;
+
+function CardCVCNumberInputs({
+  CVCNumber,
+  changeCVCNumber,
+  viewNextInput,
+  getErrorMessage,
+  isInvalid,
+}: CardCVCNumberSectionProps & ValidationProps) {
+  return (
+    <StyledContainer>
+      <label htmlFor="">CVC</label>
+      <StyledInputWrap>
+        <Input
+          value={CVCNumber}
+          onChange={(e) => {
```

Comment:
value만 뽑아서 사용해도 괜찮을거 같네용 ㅎㅎ
```suggestion
          onChange={(target: {value}) => {
```

Reply:
가독성 측면에서와 이벤트 객체임을 알기위해서는 e로 쓰는게 더 좋지 않을까 생각합니다!